### ISSUE TITLE
NTR: Vitamin B12-binding protein deficiency (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8205]
New term request Vitamin B12-binding protein deficiency (segal_080501173453): R binder protein

**Comment by pnrobinson** [comment]
@MickeySegal

Is this equivalent to [Decreased circulating transcobalamin II concentration HP:0034608](https://hpo.jax.org/app/browse/term/HP:0034608)

-- The TCN2 gene encodes transcobalamin II (TC II), a plasma globulin that acts as the primary transport protein for vitamin B12 (https://omim.org/entry/613441)

**Comment by pnrobinson** [comment]
@MickeySegal 

Adding new term: 
- ID: HP:6000967
- label: Decreased circulating transcobalamin I concentration
- parents: [Abnormal circulating protein concentration (HP:0010876)](https://hpo.jax.org/app/browse/term/HP:0010876)
- pmid: 
- def: Concentration of transcobalamin I in the blood circulation below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8205
- synonyms: Decreased circulating haptocorrin concentration(NONE)|Decreased circulating R vitamin B12 binder protein concentration(NONE)|Decreased circulating cobalophilin concentration(NONE)
